### ISSUE TITLE
thing that auto-runs a repo

### ISSUE THREAD

**Issue by adampbeardsley** [loco-lab/HackProjects#1]
I want to make a simple thing that integrates with github to auto-run a pipeline when a PR is made. Here's what I'm thinking:

- When you make a PR, there's a button to run
- When you hit the button, some service on enterprise downloads the repo and runs a script
- That service then reports back to the PR saying it ran, and where the output is
- bonus points if we can put a plot in the PR.

**Comment by mkolopanis** [comment]
you basically just invented CI and CI reporting bots

**Comment by adampbeardsley** [comment]
Yeah, but I want to own it so we can add to it. Ultimately we want to be able to run on larger data sets, store and tag outputs, and keep more logs about the environment. But this is a first step.

**Comment by mkolopanis** [comment]
also you can use github actions to ssh to a server and execute code. which is kind of this

**Comment by dannyjacobs** [comment]
so it should be do-able in a day?
